### ISSUE TITLE
Stuck in loop indexing unauthorized provider

### ISSUE THREAD

**Issue by mihaisc** [oceanprotocol/aquarius#893]
```
2022-09-23 13:21:37 | 2022-09-23 10:21:37,743 - aquarius.events.events_monitor - ERROR - Error processing event: Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0xaf824d35e416bb13090b6c3d8a9a3a3c285fc891df88ec9f6568bf9ab97fa5cc11821881aef352a6ebecfe434169535d98f7eb3a28823f83251f60d0801408b201', 'nonce': '1663928497'}, response is b'{"error":"Decrypter not authorized"}\n'.
-- | --
  |   | 2022-09-23 13:21:37 | 2022-09-23 10:21:37,742 - aquarius.events.decryptor - ERROR - Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0xaf824d35e416bb13090b6c3d8a9a3a3c285fc891df88ec9f6568bf9ab97fa5cc11821881aef352a6ebecfe434169535d98f7eb3a28823f83251f60d0801408b201', 'nonce': '1663928497'}, response is b'{"error":"Decrypter not authorized"}\n'
  |   | 2022-09-23 13:21:26 | 2022-09-23 10:21:26,257 - aquarius.events.events_monitor - ERROR - Error processing event: Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0xffc92279998401d36d9da22c7b5166d8a5ec4b0f64c6bb8933dd20cf51ee272650268a760935199b3afe6fcfe6be812c68896ae0a0904bcee4f072d27dccdef801', 'nonce': '1663928485'}, response is b'{"error":"Decrypter not authorized"}\n'.
  |   | 2022-09-23 13:21:26 | 2022-09-23 10:21:26,256 - aquarius.events.decryptor - ERROR - Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0xffc92279998401d36d9da22c7b5166d8a5ec4b0f64c6bb8933dd20cf51ee272650268a760935199b3afe6fcfe6be812c68896ae0a0904bcee4f072d27dccdef801', 'nonce': '1663928485'}, response is b'{"error":"Decrypter not authorized"}\n'
  |   | 2022-09-23 13:21:14 | 2022-09-23 10:21:14,463 - aquarius.events.events_monitor - ERROR - Error processing event: Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0x904bb5c519a31817392b73aa76f332d09e3eea716b0d09b8ca24bd01bb74106835b0fd36cb614bd7a533b91efae4fcb02eeebb9ff8afb3e8c060680278cd5a7b00', 'nonce': '1663928473'}, response is b'{"error":"Decrypter not authorized"}\n'.
  |   | 2022-09-23 13:21:14 | 2022-09-23 10:21:14,463 - aquarius.events.decryptor - ERROR - Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0x904bb5c519a31817392b73aa76f332d09e3eea716b0d09b8ca24bd01bb74106835b0fd36cb614bd7a533b91efae4fcb02eeebb9ff8afb3e8c060680278cd5a7b00', 'nonce': '1663928473'}, response is b'{"error":"Decrypter not authorized"}\n'
  |   | 2022-09-23 13:21:02 | 2022-09-23 10:21:02,900 - aquarius.events.events_monitor - ERROR - Error processing event: Failed to get a response for decrypt DDO with provider=https://v1.provider.goerli.dev.acentrik.io, payload={'transactionId': '0x38ea775ba91c1de1842169b7c507255dd3cc2d2fbaf85742d89abc9814389ebf', 'chainId': 5, 'decrypterAddress': '0xF41681F0BE4A914d447d7A94C91906E74A6561D0', 'dataNftAddress': '0xA61163E975FC6bB82360d67e5EFc9997F9EB7f02', 'signature': '0x9b2c104c14c8a6ce5865b3655aea8949578a0f929d78894204cc50e1bea794bd4c854aa63d1ed236b383b93b433c4e75ac2932639923f62afef760de5385ff2a00', 'nonce': '1663928462'}, response is b'{"error":"Decrypter not authorized"}\n'.
```

This is just a small example, it's full of this transaction. Basically it's stuck

**Comment by alexcos20** [comment]
I was afraid of this

**Comment by calina-c** [comment]
@alexcos20 this was merged to main and is part of the 4.4.2 release. Should we wait until this is deployed to any other networks before closing the ticket? I'm putting it in review until then.
